tests/Neo.Network.RPC.Tests/UT_RpcClient.cs

```diff
@@ -78,6 +79,13 @@ public void TestConstructorByUrlAndDispose()
             action.Should().NotThrow<Exception>();
         }
 
+        [TestMethod]
+        public void TestRpcServerSettingsConstructorForFormatException()
```

Comment:
I think this should not be in `UT_RpcClient.cs`.

Reply:
I thought so. Mind if I create a `UT_RpcServer.cs` and move the test there?

Reply:
I've moved the test to a separate test project. Given the current setup of the unit tests projects, I've decided to move it to a separate Neo.Plugins.RpcServer.Tests folder.